.idea/deploymentTargetDropDown.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I think the files here in `.idea` should not be commited: `deploymentTargetDropDown.xml`, `compiler.xml`, `misc.xml`.